packages/calcite-components/src/components/segmented-control-item/segmented-control-item.scss

```diff
@@ -1,79 +1,105 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-segmented-control-item-background-color: Specifies the background color of the component.
+ * @prop --calcite-segmented-control-item-border-color: Specifies the border color of the component.
+ * @prop --calcite-segmented-control-item-focus: Specifies the focus color of the component.
+ * @prop --calcite-segmented-control-item-icon-color: Specifies the icon color of the component.
+ * @prop --calcite-segmented-control-item-shadow: Specifies the shadow of the component.
+ * @prop --calcite-segmented-control-item-text-color: Specifies the text color of the component.
```

Comment:
add `--calcite-segmented-control-item-icon-color` doc here

Reply:
@alisonailea it is there.